### DISCUSSION TITLE
I read the whole post but i still cant know how to use

### DISCUSSION THREAD

**Discussion by fatxx57** [Sam5440/Genshin_Impact_Teleport#9] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/94171042/199686742-0e7d3355-64a2-441b-8456-09602d4445fc.png)
Please help me. Ty!!

**Comment by fatxx57** [comment]
how to show the table like yours

**Comment by Sam5440** [comment, marked as answer]
ask your question  in akebigc group,you can get the answer quickly.=_=

**Comment by 3415549902** [comment]
where can get the new files

**Reply by Sam5440** [reply]
ask you question in discord

**Comment by HaR1zOn** [comment]
Ok, inject cheat (akebi/akrepi/bkebi), open menu, press the button "teleport", "custom tp", "open folder", paste all the files in the folder you need, press the "reload" button and enjoy
